uk.co.compucorp.civicrm.hremails/email-templates/src/partials/request-comment.html

```diff
@@ -0,0 +1,15 @@
+<table class="request-comment">
+  <tbody>
+    <tr>
+      <td class="request-comment-header">
+        <span class="request-comment-author">{{commenter}}:</span>
+        <span class="request-comment-datetime">{{createdAt}}</span>
+      </td>
+    </tr>
+    <tr>
+      <td class="request-comment-body">
```

Comment:
`<td class="request-comment-body">{{text}}</td>` as you are doing the same in other places.